Source/WebKit/WebProcess/WebPage/UnifiedTextReplacementController.h

```diff
@@ -97,6 +98,9 @@ class UnifiedTextReplacementController final {
 
     WeakPtr<WebPage> m_webPage;
 
+    typedef std::pair<WTF::UUID, WebCore::CharacterRange> TextIndicatorCharacterRange;
```

Comment:
this should be `using TextIndicatorCharacterRange = std::pair<WTF::UUID, WebCore::CharacterRange>`